sdk/trafficmanager/Azure.ResourceManager.TrafficManager/src/Customization/TrafficManagerUserMetricsKeysRestOperations.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.ResourceManager.TrafficManager.Models;
+
+namespace Azure.ResourceManager.TrafficManager
```

Comment:
lets mirror the folder structure in the `Generated` folder.  This fille should go in a subfolder called `RestOperations`